virtme/commands/run.py

```diff
@@ -1293,28 +1300,55 @@ def do_it() -> int:
     else:
         virtme_init_cmd = "virtme-init"
 
+    initsh = []
+    if args.systemd:
+        kernelargs.append("virtme_systemd=1")
+        kernelargs.append(f"console={arch.serial_console_args()[0]}")
+        initsh.extend(
+            [
+                "mount --bind /dev/null /usr/lib/systemd/system-generators/systemd-fstab-generator",
+                "mount --bind /dev/null /usr/lib/systemd/system-generators/systemd-cryptsetup-generator",
+                "mount --bind /dev/null /usr/lib/systemd/system/systemd-journald.service",
+            ]
+        )
+
     if args.root == "/":
```

Comment:
I haven't checked in detail, but to me it seems like the only difference if args.root is set to "/" that the guest_tools_path is either `/run/virtme/guesttools` or `f"{guest_tools_path}"`.

Couldn't we use this fact and make the code simpler? (at multiple places)

Reply:
Yep, it is out-of-scope.